repotracker/repotracker.go

```diff
@@ -621,11 +621,32 @@ func CreateVersionFromConfig(ctx context.Context, projectInfo *model.ProjectInfo
 	}
 	v.Ignored = ignore
 
+	// Compute aliases first, so we can include this in the version errors if there's a problem.
+	var aliases model.ProjectAliases
+	var aliasErr string
+	if metadata.Alias == evergreen.GitTagAlias {
+		aliases, err = model.FindMatchingGitTagAliasesInProject(projectInfo.Ref.Id, metadata.GitTag.Tag)
+		if err != nil {
+			return v, errors.Wrapf(err, "error finding project alias for tag '%s'", metadata.GitTag.Tag)
+		}
+	} else if metadata.Alias != "" {
+		aliases, err = model.FindAliasInProjectRepoOrConfig(projectInfo.Ref.Id, metadata.Alias)
+		if err != nil {
+			return v, errors.Wrap(err, "error finding project alias")
+		}
+	}
```

Comment:
Could you move the `var aliasErr string` to above this check since it's not relevant to the above logic? When I read through the above logic, I expected it to be used and thought it was an error at first but then realized it wasn't